Update the gce manifest template formatting

Jonty opened the pull request:
This brings the gce manifest template formatting in line with the aws
manifest template.

Minor changes other than formatting/reordering ported from the aws template:
- The passwords have been changed to match aws ("thing-password" rather
  than "thing")
- max_threads has been specified on the director

Things I have not ported:
- The aws registry configuration is substantially larger, enabling the
  HTTP interface and configuring the database. I suspect this needs
  pulling in too.

Jonty committed:
Update the gce manifest template formatting

This brings the gce manifest template formatting in line with the aws
manifest template.

Minor changes other than formatting/reordering ported from the aws template:
* The passwords have been changed to match aws ("thing-password" rather
  than "thing")
* max_threads has been specified on the director

Things I have not ported:
* The aws registry configuration is substantially larger, enabling the
  HTTP interface and configuring the database. I suspect this needs
  pulling in too.

mtekel commented:
I started doing the same work with @actionjack when we were going ahead with CF deployment on GCE for the first time and couldn't figure out why the manifests are different and what exactly are the differences. We should have done it like you did now - which would save you solving exactly same problem. Except that story exploded into too many other things, so we didn't do this. Good to have done it now.

mtekel committed:
gce microbosh manifest: fix static ip list

Static IPs were list before - and should stay after reformatting. This looks like a forgotten "typo" from previous commit. Would squash it in, but author is different, hence making this a separate commit.

mtekel committed:
gce microbosh manifest: fix nats password

NATS password was updated to match AWS, but the password used by agent was left at old value. Fix that.

mtekel commented:
This works....
